flink-core/src/main/java/org/apache/flink/configuration/CheckpointingOptions.java

```diff
@@ -576,8 +576,8 @@ public class CheckpointingOptions {
                                     .build());
 
     @Experimental
-    public static final ConfigOption<Boolean> ENABLE_UNALIGNED_INTERRUPTIBLE_TIMERS =
-            ConfigOptions.key("execution.checkpointing.unaligned.interruptible-timers.enabled")
+    public static final ConfigOption<Boolean> ENABLE_UNALIGNED_SPLITTABLE_TIMERS =
```

Comment:
I assume existing configs will break - if they were using the old values. I know it is experimental - but can we help the user here. Maybe if they use the old value then they should get an error.  

I only see `execution.checkpointing.unaligned.interruptible-timers.enabled` defined once in master - why do you think this is a cut and paste error?

Reply:
We have an internal fork of Flink and I noticed that we only used the splittable config instead of the interruptible config. I figured that maybe something went wrong when @pnowojski open sourced this feature. Piotr can you confirm whether we should be using `splittable` or `interruptible` here,  because right now they are intermixed like on this line https://github.com/apache/flink/pull/26666/files#diff-35fa4795524db4fca98cd404f3184bf9ec2a903d673f49074a1ab970733aa0eaL1261-L1262

Reply:
maybe instead of renaming the config `execution.checkpointing.unaligned.interruptible-timers.enabled`, a smaller change would be to replace the use of `splittable` with `interruptible`, like on the line above.

Reply:
It was a conscious decision when I was open sourcing this feature. Yes, internal version was first named "splittable", but "interruptible" makes more sense. That's also what have been voted in the FLIP:

https://cwiki.apache.org/confluence/display/FLINK/FLIP-443%3A+Interruptible+timers+firing